database/src/main/resources/db/migration/V1_142__remove_old_change_dates.sql

```diff
@@ -0,0 +1,5 @@
+-- Remove gtfs-transit-changes.change-date dates from database to prevent unwanted change-detections
```

Comment:
Doesn't this affect production as well? Current week's change dates would be lost, and re-created...when?

Reply:
I assume that we will get this pr to production in the next week or two. Next correct detection cycle starts about 4 weeks from now. So if there is change-days that are in the next 7 days, they are generated with wrong code.